docs/content/administration/install.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Install Child
+description: How to get Child up and running on your network
+---
+{{< hint warning >}}
+**Current limitations**
+- Child doesn't support linking over TLS/SSL to an ircd. The current recommendation is to link to a local ircd on the local interface (127.0.0.1).
+- Child only supports MySQL 5.7
```

Comment:
How do we know that?

Reply:
I assumed it was the case, I'll remove that point.